data/json/monsters/zed_misc.json

```diff
@@ -299,6 +299,55 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_zombie_nemesis",
+    "type": "MONSTER",
+    "name": { "str": "doppelganger" },
+    "description": "A repulsive humanoid, its grotesque muscles throb with an infernal energy. In its twisted visage, which has haunted your dreams since childhood, you can see the demonic reflection of your own face.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN", "NEMESIS" ],
+    "diff": 5,
+    "volume": "875000 ml",
+    "weight": "200 kg",
+    "hp": 480,
+    "speed": 50,
```

Comment:
This seems quite slow. Maybe bump it up to 70 or 80? Still outrunnable, but not quite so trivial.